### ISSUE TITLE
Cross-talk between UI clients

### ISSUE THREAD

**Issue by leahneukirchen** [vamolessa/pepper#5]
Open one pepper and activate the command mode `:foo`, don't press return yet.

Open another pepper, it takes over the prompt now.

**Comment by vamolessa** [comment]
Yeah, it's like that because clients are dumb: they only send keys to server and receive back what to render.
So when another clients comes in, the prompt state is not lost since it's on the server.

**Comment by leahneukirchen** [comment]
But if two clients are connected, shouldn't they have their own prompt states? (It does not need to be stored locally.)

**Comment by vamolessa** [comment]
That makes sense. However I'm not sure it's an easy change as prompts are tied to the mode the editor is in.
So each client having it's own prompt would mean each client having it's own mode.
And we'd have to check the implications of it.

**Comment by vamolessa** [comment]
ops, sorry I closed accidentally
